src/main/java/uk/gov/hmcts/darts/authentication/dao/AzureDao.java

```diff
@@ -1,8 +1,11 @@
 package uk.gov.hmcts.darts.authentication.dao;
 
+import uk.gov.hmcts.darts.authentication.config.AuthConfigurationProperties;
+import uk.gov.hmcts.darts.authentication.config.AuthProviderConfigurationProperties;
 import uk.gov.hmcts.darts.authentication.exception.AzureDaoException;
 import uk.gov.hmcts.darts.authentication.model.OAuthProviderRawResponse;
 
 public interface AzureDao {
```

Comment:
Maybe good to also make this interface more generic so we remain agnostic of any paticular cloud provider e.g. idpDAO?